### DISCUSSION TITLE
Help with custom Link variants

### DISCUSSION THREAD

**Discussion by JClackett** [chakra-ui/chakra-ui#4064] [Q&A] [answered]
Hey!

I'm a bit confused about how to get custom Link variants, this is my theme:

```js
export const theme = extendTheme({
  components: {
    Link: {
      defaultProps: { variant: "test" },
      variants: {
        test: {
          p: 4,
          borderRadius: "md",
          bgColor: "red"
        }
      }
    }
  }
});
```

But the styling doesn't change when using this variant, here's a [codesandbox](https://codesandbox.io/s/fragrant-monad-2uzcs?file=/pages/_app.tsx:102-342), any ideas?

Thanks

**Comment by payapula** [comment, marked as answer]
@JClackett I checked your sandbox and seems the variant is working for me. Can you check?

![image](https://user-images.githubusercontent.com/7134153/132097920-fbd73e8e-45ca-480f-af3b-bb0f8b6c4631.png)
